[2025-07-17T10:07:09.847+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_data_flow.get_data manual__2025-07-17T10:07:08.558312+00:00 [queued]>
[2025-07-17T10:07:09.850+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_data_flow.get_data manual__2025-07-17T10:07:08.558312+00:00 [queued]>
[2025-07-17T10:07:09.850+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-07-17T10:07:09.854+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_data> on 2025-07-17 10:07:08.558312+00:00
[2025-07-17T10:07:09.855+0000] {standard_task_runner.py:60} INFO - Started process 197 to run task
[2025-07-17T10:07:09.857+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'api_data_flow', 'get_data', 'manual__2025-07-17T10:07:08.558312+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/controller_dag.py', '--cfg-path', '/tmp/tmpbiwm6zoo']
[2025-07-17T10:07:09.857+0000] {standard_task_runner.py:88} INFO - Job 5: Subtask get_data
[2025-07-17T10:07:09.864+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-17T10:07:09.877+0000] {task_command.py:423} INFO - Running <TaskInstance: api_data_flow.get_data manual__2025-07-17T10:07:08.558312+00:00 [running]> on host 7983701cd435
[2025-07-17T10:07:09.908+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='api_data_flow' AIRFLOW_CTX_TASK_ID='get_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-17T10:07:08.558312+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-17T10:07:08.558312+00:00'
[2025-07-17T10:07:09.920+0000] {logging_mixin.py:188} INFO - GET: [{'id': 1, 'name': 'Kotlin'}, {'id': 2, 'name': 'Spring Boot'}, {'id': 3, 'name': 'DevOps'}, {'id': 4, 'name': 'H2 Database'}, {'id': 5, 'name': 'JUnit Test'}, {'id': 6, 'name': 'data-1'}, {'id': 7, 'name': 'data-2'}, {'id': 8, 'name': 'data-3'}, {'id': 9, 'name': 'data-1'}, {'id': 10, 'name': 'data-2'}, {'id': 11, 'name': 'data-3'}]
[2025-07-17T10:07:09.920+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-07-17T10:07:09.925+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=api_data_flow, task_id=get_data, execution_date=20250717T100708, start_date=20250717T100709, end_date=20250717T100709
[2025-07-17T10:07:09.958+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-07-17T10:07:09.966+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
